recipe/meta.yaml

```diff
@@ -10,30 +10,42 @@ source:
   sha256: 3be9ca9d812e760969419e7288fef4d6700e7aa01d303aad4e56cd4ecb4f11ed
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
   # This packages uses libhdfs3, which is primarily used with HDFS on Linux.
-  skip: True  # [win]
+  # There is no libhdfs3 >=2.3 and its dependencies on s390x
+  skip: True  # [win or osx or s390x]
 
 requirements:
   host:
     - python
+    - pip
     - setuptools
-
+    - wheel
   run:
     - python
     - libhdfs3 >=2.3
 
 test:
   imports:
     - hdfs3
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://github.com/dask/hdfs3
-  license: BSD 3-Clause
+  license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE.txt
   summary: Python wrapper for libhdfs3
+  # This project is not undergoing development.
+  # Pyarrow's JNI hdfs interface is mature and stable. 
+  # It also has fewer problems with configuration and various security settings, 
+  # and does not require the complex build process of libhdfs3. 
+  # Therefore, all users who have trouble with hdfs3 are recommended to try pyarrow.
+  dev_url: https://github.com/dask/hdfs3
 
```

Comment:
We might have too add the `doc_url:`

Reply:
Thanks. I added `doc_url: https://hdfs3.readthedocs.io/en/latest/`